packages/components/app/styles/components/dropdown.scss

```diff
@@ -156,171 +162,208 @@ $size-props: (
 }
 
 
-// TODO dropdown container, should this be styled here?
-.hds-dropdown__list {
-  align-items: flex-start;
-  background-color: white; //TODO REPLACE WITH TOKEN
+// LIST
+// UL ELEMENT
+// GOES INSIDE HDS::DISCLOSURE's :content block
+
+.hds-dropdown-list {
+  background-color: var(--token-color-surface-primary);
   border-radius: 6px;
-  // box-shadow: var(--token-elevation-high-box-shadow); //TODO what should this actually be?
-  box-shadow: 0px 0px 0px 1px rgba(101, 106, 118, 0.25), 0px 2px 3px rgba(101, 106, 118, 0.15), 0px 16px 16px -10px rgba(101, 106, 118, 0.2);
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+  box-shadow: var(--token-surface-high-box-shadow);
+  box-sizing: border-box;  //TODO 
   list-style: none;
-  margin: 0;
-  min-height: fit-content;
+  margin: 6px 0 0;
   padding: 4px 0;
-  width: 200px; // default width
+  width: 200px; // default width // TODO
 
   &.absolute--right {
     position: absolute;
     right: 0;
+    z-index: 2; // TODO understand what is the right value here
   }
 
   &.absolute--left {
     position: absolute;
     left: 0;
+    z-index: 2; // TODO understand what is the right value here
   }
+}
 
-  .hds-dropdown-list-item {
-    align-items: center;
-    box-sizing: border-box;
-    display: flex;
-    font-family: var(--token-typography-font-stack-text);
-    min-height: 36px;
-    justify-content: center;
+
+// LIST > LIST-ITEM
+// HDS::DROPDOWN::LIST-ITEM
+
+.hds-dropdown-list-item {
+  // TODO flex removed? (MS)
+
+  &.hds-dropdown-list-item--heading {
```

Comment:
[issue] you don't need to nest these declarations, they're are already name-spaced, so there's no risk of naming conflicts (plus, the specificity increases)

Reply:
done